lib/process-services-cloud/src/lib/services/form-fields.interfaces.ts

```diff
@@ -26,6 +26,7 @@ export interface FormRepresentation {
     version?: number;
     formDefinition?: FormDefinition;
     standAlone?: boolean;
+    forceFullScreen?: boolean;
```

Comment:
I think we could be more generic, and introduce the `displayMode` property here.
`default` would be our "legacy" display of the form, and now we could add `fullscreen`. Later on this way, we can enrich display modes.

Reply:
So you mean that we can introduce here instead of a `boolean` an enumerated value, and by now, we only have two options here `inline`(default) and `fullScreen`. I think that I will do something like this to be more generic, so the components receive an array of available display mode configurations, and the default display mode:

```typescript
export enum FormCloudDisplayMode {
    inline = 'inline',
    fullScreen = 'fullScreen'
}

export interface FormCloudDisplayModeConfiguration: {
	displayMode: FormCloudDisplayMode,
	options: any
};
`
``

I will create method in the `FormRenderingService` for getting the default display model configuration if not provided.

OK this makes sense, I will use a radio button group in the modeling for selecting the `displayMode` in the form properties